cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/ConnectionNotification/ConnectionNotification.tsx

```diff
@@ -0,0 +1,41 @@
+// React imports
+import React from 'react';
+// Library imports
+import { CdsAlert, CdsAlertGroup } from '@cds/react/alert';
+// App imports
+import './ConnectionNotification.scss';
+
+export function ConnectionNotification(
+    connected: boolean,
+    connectedMessage: string,
+    connecting: boolean,
+    connectingMessage: string,
+    errorMessage?: string
+) {
+    return (
+        <div>
+            {connected && (
+                <CdsAlertGroup status="success" aria-label={connectedMessage}>
+                    <CdsAlert>{connectedMessage}</CdsAlert>
+                </CdsAlertGroup>
+            )}
+            {connecting && (
+                <CdsAlertGroup status="loading" aria-label={connectedMessage}>
+                    <CdsAlert>{connectingMessage}</CdsAlert>
+                </CdsAlertGroup>
+            )}
+            {!connected && errorMessage && (
+                <CdsAlertGroup status="danger">
+                    <CdsAlert>{errorMessage}</CdsAlert>
+                </CdsAlertGroup>
+            )}
+            {!connected && !connecting && !errorMessage && (
+                <div className="hide-me">
+                    <CdsAlertGroup status="neutral">
+                        <CdsAlert>&nbsp;</CdsAlert>
```

Comment:
what is the space &nbsp; for?

Reply:
The idea is to have an "empty" or hidden alert taking up the space so that the screen doesn't "jump" when the message displays. If there's a better way to do it, I'm happy to change it.